### ISSUE TITLE
The time in the log is not displayed correctly.

### ISSUE THREAD

**Issue by fizsef** [AdguardTeam/AdGuardHome#5587]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to report a bug and not ask a question

### Operating system type

Linux, OpenWrt

### CPU architecture

32-bit MIPS

### Installation

GitHub releases or script from README

### Setup

On one machine

### AdGuard Home version

v0.107.26

#### What did you do?
I want to see the local time in the log. Enabled the "log_localtime" option.
#### Expected result
The time in the log should correspond to the time on the router.
#### Actual result
The time in the log does NOT match the time on the router.
#### Screenshots (if applicable)
![1](https://user-images.githubusercontent.com/74646605/224531053-944ee56e-4939-4646-b8d1-b6cc4a580512.jpg)
![2](https://user-images.githubusercontent.com/74646605/224531054-77c85d29-cfc6-4643-ae4d-5cf25047a1b7.jpg)

**Comment by fizsef** [comment]
Well, I'm talking about the log file (text file) of the AGH itself, and not about the query log. It can be seen in the screenshot above that my question was about the log file. I have it located on the path /tmp/AdGuard Home.log

**Comment by ainar-g** [comment]
Right, sorry.  In that case, this is likely due to AdGuard Home not getting the time zone from the environment.  In particular, the `TZ` environment variable.

See #4207 and #3482.
